addons/mass_mailing/wizard/mailing_mailing_schedule_date.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MailingMailingScheduleDate(models.TransientModel):
+    _name = "mailing.mailing.schedule.date"
+    _description = "schedule a mailing"
+
+    schedule_date = fields.Datetime(string='Scheduled for')
+    mass_mailing_id = fields.Many2one('mailing.mailing', required=True)
+
+    def set_schedule_date(self):
```

Comment:
Guidelines: as this is an action called from a form view, we generally prefix with "action", like "action_schedule" .

Reply:
Changed to action_schedule_date.